text/0000-sawtooth-sdk-swift.md

```diff
@@ -0,0 +1,214 @@
+SDKs and Swift will be able to use the Swift SDK with minimal learning
+friction.
+
+#### Objective-C SDK
+
+An SDK that supports Cocoa/Cocoa Touch Applications could also be written in
+Objective-C. In spite of Objective-C being an older language with many
+feature-rich APIs, Swift is a better choice for a couple of reasons:
+
+- Swift is an language that is easier to learn, with simpler syntax which 
+  makes code easier to read and write. 
+- Using Swift makes is possible to export the SawtoothSigning module as a
+  dynamic framework rather than a static library. Dynamic frameworks reduce
+  the launch time of apps and their memory footprints when compared to static
+  libraries. To use SawtoothSigning as dynamic framework also allows the users
+  to use Carthage to download and build the module directly from the GitHub
+  repository.
+
+#### Use existing functionality
+
+Without the Swift SDK, developers have two options to write an iOS-compatible 
+Sawtooth client:
+- Create a web app using the Javascript SDK
+- Implement custom methods to create and sign transactions natively
+
+The first solution does not work for a use case that requires a native
+application. The second solution raises the barrier of entry for developers
+with a background in mobile development who want to use Sawtooth. The creation
+of the Sawtooth Swift SDK, including documentation and examples, will make the
+Sawtooth platform more accessible to those coming from mobile development.
+
+# Prior art
+[prior-art]: #prior-art
+
+The implementation of the SawtoothSigning module for the Swift SDK is based
+on existing implementations for other languages, such as 
+[Java](https://github.com/hyperledger/sawtooth-sdk-java/tree/master/sawtooth-sdk-signing/src/main/java/sawtooth/sdk/signing) 
+and
+[Rust](https://github.com/hyperledger/sawtooth-sdk-rust/blob/master/src/signing/secp256k1.rs).
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+### To be implemented
+
+- To use Carthage to import the SawtoothSigning framework into an iOS project
+  it is necessary to include a Cartfile, which describes the project’s
+  dependencies to Carthage. GitHub repositories are specified with the github
+  keyword. For example, it is possible to load the framework from a branch:
+  ```
+  github "bitwiseio/sawtooth-sdk-swift" "master"
+  ```
+
+  Or from a release:
+  ```
+  github "bitwiseio/sawtooth-sdk-swift" ~> 0.1
+  ```
+
+  If a new repository within the Hyperledger organization is created to host
+  the Sawtooth Swift SDK, the existing Carfile in the XO example needs to be
+  updated for the new path of the SDK. The documentation also needs to be
+  updated to include instructions on how to import the SawtoothSigning
+  framework from the new hyperledger repository.
+
+- The Sawtooth protos are not included as part of the Swift SDK. Before
+  stabilization, we may want to include them as part of the SDK. Currently,
```

Comment:
(Nitpick) A bit confusing, grammatically - it implies that we are not yet stabilized, rather than the protos.  I suggest moving/rewording "before stabilization" to later in the sentence, so that there's no hint of a dangling modifier.  How about: 

"We might want to include them temporarily until the Swift SDK is stabilized."

Reply:
Updated this description